service/controller/v18/key/key.go

```diff
@@ -103,7 +103,13 @@ func AutoScalingGroupName(customObject v1alpha1.AWSConfig, groupName string) str
 }
 
 func AvailabilityZone(customObject v1alpha1.AWSConfig) string {
-	return customObject.Spec.AWS.AZ
+	availabilityZones := customObject.AvailabilityZones()
+
+	if len(availabilityZones) == 0 {
+		return ""
+	}
+
+	return availabilityZones[0]
```

Comment:
This will change again with multiple AZs I assume.

Reply:
Oh yes. But I just want to get going with it.